.gitlab/e2e.yml

```
@@ -32,9 +39,6 @@ k8s-e2e-cws-dev:
 k8s-e2e-cws-main:
   extends: .k8s_e2e_template
   rules: !reference [.on_main]
-  # needs:
-  #   - dev_master-a6
-  #   - dev_master-a7
   retry: 1
   script:
     - !reference [.k8s-e2e-cws-cspm-init]
```

This job is using
* `datadog/agent-dev:master-py3`
* `datadog/cluster-agent-dev:master`

that are published by:

```suggestion
k8s-e2e-cws-main:
  extends: .k8s_e2e_template
  rules: !reference [.on_main]
  needs:
    - dev_master-a7
    - dca_dev_master
  retry: 1
  script:
    - !reference [.k8s-e2e-cws-cspm-init]
```

See https://github.com/DataDog/datadog-agent/blob/0be86ff70fe5748cac248b53f90b4cd8e5eba150/.gitlab/dev_container_deploy/docker_linux.yml#L120 and https://github.com/DataDog/datadog-agent/blob/0be86ff70fe5748cac248b53f90b4cd8e5eba150/.gitlab/dev_container_deploy/docker_linux.yml#L168